src/components/ButtonGroup/ButtonGroup.js

```diff
@@ -7,17 +7,33 @@ import { directions } from '../../styles/constants';
 
 const ALIGMENT_TYPES = [directions.LEFT, directions.CENTER, directions.RIGHT];
 
-const marginStyles = ({ theme, noMargin }) =>
-  !noMargin &&
-  css`
-    li:not(:last-of-type) {
+const getInlineStyles = theme => css`
```

Comment:
This whole diff is wrong. We merged the PR updating the `ButtonGroup` stuff already, haven't we? Here's the closed PR for this change: https://github.com/sumup/circuit-ui/pull/246

Why is this showing up in the diff now? 😨